examples/llm_compression/openvino/smollm2_360m_fp8/requirements.txt

```diff
@@ -2,4 +2,4 @@ datasets
 openvino==2024.5
 optimum-intel[openvino]
 transformers
-onnx<1.16.2
+onnx==1.17.0
```

Comment:
I think it can be removed now, right?

Reply:
Every example has this requirement. If we remove it here, do not we need to remove it elsewhere? @AlexanderDokuchaev

Reply:
@MaximProshin 
It's part of our styleguide https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md#65-bkc-management
Need to avoid problem this hidden updating version of packages in tests, and know what we are testing.

Reply:
Then I need to understand why we have onnx requirement in LLM examples specifically. Can you explain please?

Reply:
@MaximProshin let's keep it as is